plugins/database/redshift/redshift_test.go

```diff
@@ -421,15 +467,55 @@ func TestPostgreSQL_RotateRootCredentials(t *testing.T) {
 		t.Fatal("Database should be initialized")
 	}
 
-	newConf, err := db.RotateRootCredentials(context.Background(), nil)
+	const tempPassword = "MyTemporaryRootPassword1!"
+	updateReq := newdbplugin.UpdateUserRequest{
+		Username: adminUser,
+		Password: &newdbplugin.ChangePassword{
+			NewPassword: tempPassword,
+		},
+	}
+	_, err = db.UpdateUser(context.Background(), updateReq)
 	if err != nil {
 		t.Fatalf("err: %v", err)
 	}
 
-	fmt.Printf("rotated root credentials, new user/pass:\nusername: %s\npassword: %s\n", newConf["username"], newConf["password"])
+	if err = testCredsExist(t, url, adminUser, tempPassword); err != nil {
+		t.Fatalf("Failed to test new admin user credentials after rotation: %s", err)
+	}
+
+	err = db.Close()
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+
+	// Switch it back at the end to make the test somewhat less destructive.
+	// We re-initialize the whole plugin, which is roughly what the real database
```

Comment:
Is this comment correct? This was my conclusion from taking a look through the backend, when I was trying to figure out how the plugin handled creating new connections after a root credential change.